README.md

```diff
@@ -8,9 +8,9 @@ Let's imagine you have a fruit store.
 There are three type of operations in your fruit store:
 
 ```text
-    s - supply, means you are receiving a new fruits from suppliers
+    s - supply, means you are receiving a new fruitStorages from suppliers
```

Comment:
i don't think it's quite correct, because we receive fruits not storages here, same goes to other changes in readme file